lte/protos/pipelined.proto

```diff
@@ -89,6 +89,7 @@ message DeactivateFlowsRequest {
   RequestOriginType request_origin = 3;
   string ip_addr = 4; // Subscriber session ipv4 address
   bytes ipv6_addr = 5; // Subscriber session ipv6 address
+  bool remove_default_drop_flows = 6;
```

Comment:
we will add this flag to `true` on our `termination` request from sessiond.
 
Other calls to deactivate flows which do not imply termination of a session rules will have this flag set to `false`.